wallet_ui/components/panels/CreateCanister.tsx

```diff
@@ -99,19 +133,30 @@ export function CreateCanisterDialog(props: {
   function create() {
     setLoading(true);
 
-    Wallet.create_canister({
-      controller: controller ? Principal.fromText(controller) : undefined,
-      cycles,
-    }).then(
-      (canisterId) => {
+    let additional = moreControllers.filter(ea => ea.length !== 0);
+    let allControllers = [controller, ...additional];
+    let result = allControllers.map(ea => Principal.fromText(ea));
+    const args = {result, cycles};
```

Comment:
Formatting nit - for consistency, we use Prettier to make sure that spacing is handled automatically. We don't have it running automatically in CI for this repo, but you can install the extension in VSCode, and set it up to format your code on save